back-end/legeno-around-here/src/main/java/wooteco/team/ittabi/legenoaroundhere/utils/MailHandler.java

```diff
@@ -0,0 +1,44 @@
+package wooteco.team.ittabi.legenoaroundhere.utils;
+
+import java.io.UnsupportedEncodingException;
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+
+public class MailHandler {
+
+    private JavaMailSender javaMailSender;
+    private MimeMessage mimeMessage;
+    private MimeMessageHelper mimeMessageHelper;
+
+    public MailHandler(JavaMailSender javaMailSender) throws MessagingException {
+        this.javaMailSender = javaMailSender;
+        mimeMessage = javaMailSender.createMimeMessage();
+        mimeMessageHelper = new MimeMessageHelper(mimeMessage, true, "UTF-8");
+    }
```

Comment:
통일감..?
```suggestion
    public MailHandler(JavaMailSender javaMailSender) throws MessagingException {
        this.javaMailSender = javaMailSender;
        this.mimeMessage = javaMailSender.createMimeMessage();
        this.mimeMessageHelper = new MimeMessageHelper(mimeMessage, true, "UTF-8");
    }
```

UTF-8은 상수로 빼도 좋겠네요~!